docs/content/how-to/powervs/create-cluster-powervs.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Create PowerVS Hypershift Cluster
+---
+
+# Create PowerVS Hypershift Cluster
+
+Creating Hypershift cluster in IBM Cloud PowerVS service.
+
+## Prerequisites
+
+Please see [prerequisites](./prerequisites-and-env-guide.md/#prerequisites) before setting up the cluster
+
+## Creating the Cluster
+
+Use the `hypershift create cluster powervs` command:
+
+    ./bin/hypershift create cluster powervs --base-domain BASEDOMAIN
+        --resource-group RESOURCE_GROUP \
+        --pull-secret PULL_SECRET \
+        --region POWERVS_REGION \
+        --zone POWERVS_ZONE \
+        --vpc-region VPC_REGION \
+        --node-pool-replicas=2
+
+E.g.:
+
+      ./bin/hypershift create cluster powervs --base-domain scnl-ibm.com \
+      --resource-group hypershift-resource-group \
+      --pull-secret ./pull-secret \
+      --region tok \
+      --zone tok04 \
+      --vpc-region jp-tok \
+      --node-pool-replicas=2 \
+
+where
+
+* BASEDOMAIN is the CIS base domain that will be used for your hosted cluster's ingress. It should be an existing CIS domain name.
+* RESOURCE_GROUP is the resource group in IBMCloud where your infrastructure resources will be created.
+* PULL_SECRET is a file that contains a valid OpenShift pull secret.
+* POWERVS_REGION is the region where you want to create the powervs resources.
+* POWERVS_ZONE is the zone under POWERVS_REGION where you want to create the powervs resources.
+* VPC_REGION is the region where you want to create the vpc resources.
+* node-pool-replicas is worker node count 
+
+Running this command will create [infra](./create-infra-powervs-separately.md/#powevs-cluster-infra-resources ) for the Hypershift cluster and will create HostedCluster and NodePool spec and deploys it.
+
+You can create infra separately and use it to create Hypershift cluster which reduces the infra creation time.
+
+See this [section](./create-infra-powervs-separately.md/#creating-the-powervs-infra) on creating the infra for Hypershift.
+
+On creating this will have a `OUTPUT_INFRA_FILE` contains details of all the resources created.
```

Comment:
I think these three lines should be moved to the following section, and we should add the `create infra` CLI step so it's clear how to generate the `OUTPUT_INFRA_FILE`?

Reply:
In previous line I have mentioned, 
`You can create infra separately and use it to create Hypershift cluster which reduces the infra creation time.`
I need to give a link to the section where infra creation separately described. Also wanted to give extra info for the user on `OUTPUT_INFRA_FILE` since that is required for the following command mentioned in 55.

Reply:
Ack, my main point was that everything related to creating infra separately should go into the `Create cluster with infra created separately` section below :)

Reply:
Makes sense, moved that part to the `create-infra-powervs-separately.md` page itself. 
Ptal. Will merge the commits if it is ok.